sqlalchemy_exasol/pyodbc.py

```diff
@@ -164,5 +167,139 @@ def is_disconnect(self, e, connection, cursor):
 
         return super().is_disconnect(e, connection, cursor)
 
+    @staticmethod
+    def _is_sql_fallback_requested(**kwargs):
+        is_fallback_requested = kwargs.get("use_sql_fallback", False)
+        if is_fallback_requested:
+            logger.warning("Using sql fallback instead of odbc functions")
+        return is_fallback_requested
+
+    @reflection.cache
+    def _tables_for_schema(self, connection, schema, table_type=None, table_name=None):
```

Comment:
_get_tables_for_schema or the name intended and if why?

Reply:
was intended that way, don't think it adds beneficial information to this non public method.